logic app templates/audit-events-template.json

```diff
@@ -0,0 +1,181 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "Logic App Name": {
+      "defaultValue": "parse-dfp-audit-events-template",
+      "type": "String"
+    }
+  },
+  "variables": {},
+  "resources": [
+    {
+      "type": "Microsoft.Logic/workflows",
+      "apiVersion": "2017-07-01",
+      "name": "[parameters('Logic App Name')]",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "state": "Enabled",
+        "definition": {
+          "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+          "contentVersion": "1.0.0.0",
+          "parameters": {
+            "$connections": {
+              "defaultValue": {},
+              "type": "Object"
+            }
+          },
+          "triggers": {
+            "When_events_are_available_in_Event_Hub": {
+              "recurrence": {
+                "frequency": "Minute",
+                "interval": 1
+              },
+              "splitOn": "@triggerBody()",
+              "type": "ApiConnection",
+              "inputs": {
+                "host": {
+                  "connection": {
+                    "name": "@parameters('$connections')['eventhubs']['connectionId']"
+                  }
+                },
+                "method": "get",
+                "path": "[concat('/', '<Your Event Hub Name Here>', '/events/batch/head')]",
```

Comment:
This part I don't believe is causing the issue because on my end it is deploying correctly and allows you to type in the Event Hub Name / select it after wards in the Logic Apps designer.